__tests__/app.test.js

```diff
@@ -51,19 +50,62 @@ describe('/api/articles/:article_id', () => {
         .get('/api/articles/1')
         .expect(200)
         .then((response) => {
-            const {rows} = response.body
-            rows.forEach(row => {
-                expect(row).toMatchObject({
+            const {article} = response.body
+            expect(article.article_id).toBe(1)
+            expect(article).toMatchObject({
+                title: expect.any(String),
+                topic: expect.any(String),
+                author: expect.any(String),
+                body: expect.any(String),
+                created_at: expect.any(String),
+                votes: expect.any(Number),
+                article_img_url: expect.any(String),
```

Comment:
You shouldn't be expecting any string or numbers on this, you know the data as you own the database. The only one where I wouldn't mind expecting any string is the created at as the db is constantly re seeded.